Makefile

```diff
@@ -38,10 +38,10 @@ GIT_REPO_ENTRYPOINT ?= collab-platform-arch.aird
 GIT_REPO_DEPTH ?= 0
 
 # T4C repository name for the importer, e.g. repoCapella
-T4C_IMPORTER_REPO ?= repo-name
+T4C_IMPORTER_REPO ?= repoCapella
```

Comment:
Comment for the complete file:
Replace tabs by spaces. Google it: "Tabs are evil" :-)

Reply:
Interesting. My VSCOde is configured to use spaces, probably something from really old times :D

Reply:
Learned something new: Makefile does only support tab characters. When using whitespaces, you get an error message: 
```
Recipes in makefile rules must start with a tab (per definition)
```